### DISCUSSION TITLE
Command not found

### DISCUSSION THREAD

**Discussion by raymie258** [seriousm4x/UpSnap#551] [Help wanted] [answered]
I uploaded the upsnap file to my home directory on a debian arm64 system. I try to run the command "./upsnap serve" and I keep getting the error command not found. I know it's probably something simple, but I can't figure it out. Any help is appreciated. Thank you

**Comment by seriousm4x** [comment, marked as answer]
You probably just need to make the file executable. `chmod +x ./upsnap` And then try again.

**Reply by raymie258** [reply]
You were so right. I'm embarrassed. TY for the program though!!!!!!
